app/views/users/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Users#index</h1>
-<p>Here is a list of users</p>
+<ul class="w-full max-w-4xl mx-auto p-6 space-y-6">
+  <% @users.each do |user| %>
+
+  
+    <li class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+    <div class="w-20 mr-2">       <%= image_tag 'img2.jpg', alt: 'icon'%>
+</div>
```

Comment:
Hi @topeogunleye, You can use it wherever you want, it can either be in the index or show.html.erb file the example requirement in our case, you can use it in the `/users/index.html.erb` Please check this link [how to use Partials](https://riptutorial.com/ruby-on-rails/example/2875/partials) for more documentation